CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2025-02-20
+
+Monitor the release status by region at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled `v20250220`.
+
+### Announcements
+* 
+
+### Release Notes
+* Features:
+   * AKS Kubernetes patch versions 1.29.12, 1.29.13, 1.30.8, 1.30.9, 1.31.4, and 1.31.5 are now available.
+   * 
+
+
+* Preview Features:
+   * AKS Kubernetes version 1.32 is now in preview.
+   * [Kubelet serving certificate rotation](https://learn.microsoft.com/Azure/AKS/certificate-rotation) now allows AKS to utilize kubelet server TLS bootstrapping for both bootstrapping and rotating serving certificates signed by the Cluster CA.
+   * [Azure Linux v3](https://learn.microsoft.com/azure/azure-linux/) is now in preview for AKS 1.32 clusters.
+
+
+* Bug Fixes:
+   * Updated the Istio service mesh add-on to allow multiple operations with the Lua EnvoyFilter.
+   *  
+
+
+* Behavior Changes:
+   * You can now configure the [default ingress controller type](https://learn.microsoft.com/en-us/azure/aks/app-routing) in the AKS application routing add-on.
+   * AKS no longer supports the [GPU image (preview)](https://github.com/Azure/AKS/issues/4472) to provision GPU-enabled AKS nodes. Alternative options that are supported today and recommended by AKS include the default experience with manual NVIDIA device plugin installation or the NVIDIA GPU Operator, detailed in [AKS GPU node pool documentation](https://learn.microsoft.com/azure/aks/gpu-cluster?tabs=add-ubuntu-gpu-node-pool#confirm-that-gpus-are-schedulable).
+   * Starting in the Karpenter v0.6 image, Node Auto Provisioning (NAP) will bootstrap nodes from Scriptless to AgentBaker using custom data provided by Node Provisioning Service (NPS) and improve the stability of NAP.
+
+* Component Updates:
```

Comment:
Please include Windows, Ubuntu, and AzLinux vhd release notes according to the instructions here: https://msazure.visualstudio.com/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/21515/Writing-weekly-AKS-release-notes?anchor=vhd-images)